cluster_tools/pyproject.toml

```diff
@@ -14,19 +14,72 @@ kubernetes = "^27.2.0"
 typing-extensions = "^4.4.0"
 
 [tool.poetry.dev-dependencies]
-black = "^23.1.0"
 icecream = "^2.1.1"
-isort = "^5.9.3"
 mypy = "^1.0.0"
-pylint = "^2.16.2"
 pytest = "^7.2.1"
+ruff = "^0.3.2"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
-[tool.isort]
-profile = "black"
+[tool.ruff]
```

Comment:
maybe add a comment that links to webknossos/pyproject.toml (and vice versa), so that it's clear that potential future changes should be done at both places.